src/server/server_family.h

```diff
@@ -35,6 +35,7 @@ struct Metrics {
   SliceEvents events;
   TieredStats tiered_stats;
   EngineShard::Stats shard_stats;
+  EngineShard::DefragStats defrag_stats;
```

Comment:
I do not see why it should be separate from EngineShard::Stats, I would move it there since these are part of the shard stats

Reply:
OK

Reply:
remove this type completely (it would require small change in unit test as well).